course_gather/settings/base.py

```diff
@@ -50,9 +50,9 @@
 ]
 
 MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
     'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
We should add Django's Gzip middleware to compress API responses as well.
https://docs.djangoproject.com/en/3.0/ref/middleware/#module-django.middleware.gzip

Reply:
Fixed in 68d321f.

Reply:
These shouldn't interfere at all. Whitenoise only handles zipping of static files. Gzip middleware is going to handle the zipping of API responses.

Reply:
👍 you're right. Fixed in 6de08e3.